src/less-style/lib/variables.less

```diff
@@ -11,3 +13,38 @@
 @grayDarker:            #2C2C2C;
 @blueLight:             #2787d0;
 
+// Main Colors [NEW]
```

Comment:
@biyeun after grabbing a new version of [hqstyle-core.css](https://github.com/dimagi/Vellum/blob/case-mgt-mockup/tests/lib/hqstyle-core.css) it looks like these colors are not in HQ yet. Do I need to wait until HQ gets some style updates to roll this PR out?